packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -286,6 +287,13 @@ class WebView extends StatefulWidget {
   /// The default policy is [AutoMediaPlaybackPolicy.require_user_action_for_all_media_types].
   final AutoMediaPlaybackPolicy initialMediaPlaybackPolicy;
 
+  /// The background color of the webview.
+  ///
+  /// When null the platform's webview default background color is used.
+  ///
+  /// By default `backgroundColor` is null.
```

Comment:
nit: documentation can be improved a little bit (similar to the Android [specific version](https://github.com/flutter/plugins/blob/7d9cc848dfd356ecc8c456b1d451d8c92a0fa507/packages/webview_flutter/webview_flutter_android/example/lib/web_view.dart#L240)).

```suggestion
  /// The background color of the [WebView].
  ///
  /// When `null` the platform's webview default background color is used. By
  /// default [backgroundColor] is `null`.
```